docs/docs/apis/popup.md

```diff
@@ -47,7 +47,8 @@ interface PopupOptions {
     // with button and chevron icon.
     getElementTriggeringPopup?: () => React.Component<any, any>;
 
-    // Called when the popup is dismissed.
+    // Called when the popup is dismissed. Popup.isDisplayed() will return
```

Comment:
Or maybe it would be safer to say that the effect of calling Popup API from within this callback is undefined.